Gemfile.lock

```diff
@@ -223,4 +223,4 @@ DEPENDENCIES
   web-console (= 3.7.0)
 
 BUNDLED WITH
-   1.17.3
+   1.17.3
```

Comment:
This file shouldn't be manually edited. There should be a new line at the end of each file.